src/plugins/orangehrmAuthenticationPlugin/Service/PasswordStrengthService.php

```diff
@@ -162,8 +168,15 @@ private function checkSpacesInPassword(string $password): ?string
      */
     private function checkRequiredDefaultPasswordStrength(int $passwordStrength): ?string
     {
-        if ($this->getConfigService()->getConfigDao()->getValue(ConfigService::KEY_DEFAULT_PASSWORD_STRENGTH)
-            > $passwordStrength
+        $defaultPasswordStrength = $this->getConfigService()->getConfigDao()->getValue(
+            ConfigService::KEY_DEFAULT_PASSWORD_STRENGTH
+        );
+        if (($defaultPasswordStrength === 'very weak' && $passwordStrength < 0)
+            || ($defaultPasswordStrength === 'weak'
+                && $passwordStrength < 1)
+            || ($defaultPasswordStrength === 'better' && $passwordStrength < 2)
+            || ($defaultPasswordStrength === 'strong' && $passwordStrength < 3)
+            || ($defaultPasswordStrength === 'strongest' && $passwordStrength < 4)
```

Comment:
@RajithaKumara is this ok or do we need to keep those values(very weak, weak ....) somewhere else.